cluster/buckets.go

```diff
@@ -43,14 +43,16 @@ const (
 	BucketCustom
 )
 
+var ErrInvalidBucketName = errors.New("invalid bucket name")
+
 // MakeBucket will get the credentials for a given tenant and use the operator keys to create a bucket using minio-go
 // TODO: allow to spcify the user performing the action (like in the API/gRPC case)
 func MakeBucket(ctx *Context, tenantShortname, bucketName string, accessType BucketAccess) error {
 	// validate bucket name
 	if bucketName != "" {
 		var re = regexp.MustCompile(`^[a-z0-9-]{3,}$`)
```

Comment:
this need to change since this is not the proper rule for bucket names

Reply:
you mean the rule?

Reply:
yes, https://docs.aws.amazon.com/AmazonS3/latest/dev/BucketRestrictions.html